Nagstamon/Config.py

```diff
@@ -179,20 +179,20 @@ def __init__(self):
 
         # Parse the command line
         parser = argparse.ArgumentParser(description='Nagios status monitor for your desktop')
-        parser.add_argument('cfgpath', nargs='*', help='Path for configuration folder')
+        parser.add_argument('cfgpath', nargs='?', help='Path for configuration folder')
```

Comment:
You used nargs='*' and then config[0] as a workaround for not adding a '-psn' argument. This is supposed to work the same. The build-script related part should be only the known_args() call.